### DISCUSSION TITLE
Uncap Baseline Minting

### DISCUSSION THREAD

**Discussion by jnthnvctr** [filecoin-project/FIPs#1031] [Q&A]
# TLDR
- Filecoin is a thriving ecosystem that has an exciting growth trajectory ahead. However, **FIL’s current tokenomics, in particular its high FDV relative to MC, are perceived to be challenging for new investors** to enter into the ecosystem. 

- With the aim of making Filecoin more aligned with how the market assesses the attractiveness of crypto networks, **we propose uncapping 770M of Baseline Minting supply**.

- If the proposal were to be accepted and implemented alongside the burning of the mining reserve, **FIL’s FDV would no longer anchor to 2b, rather to the known supply and daily inflation**. 

# Motivation
Please refer to the above [proposal](https://github.com/filecoin-project/community/discussions/708) to burn the Mining Reserve for context. 

# Proposal: Uncap the Baseline Minting Reserve

Other networks have successfully educated the market about having an algorithmically defined supply (ala Ethereum, Solana) - and it seems to have been well received. The proposal here is to do something similar for Filecoin - where rather than "capping" the supply at 1YiB (and 770m) - we extend the baseline minting curve to infinity (based on the annual 200% growth hurdle for baseline minting). Note that the actual supply would be governed by the rate of onboarding and renewals, which can be simulated [here](https://filminting.streamlit.app/Minting_Exploration). 

In this way, there is no total supply (and a high FDV) for the market to focus on - directing focus on understanding the inflation dynamics and the net effect after locking / burning. We also do not create any impacts to existing business models. 

Based on the many folks' feedback from the investment community, it is clear that the large disparity between MC and FDV has been a point of concern for prospective investors. Uncapping baseline minting and burning the Mining Reserve (see separate proposal on this), if implemented, together would materially improve the optics of FIL’s tokenomics. While this is not a silver bullet, it may serve as a good starting point for shifting the market focus from concerns around a high FDV to other more economically relevant metrics.

<img width="394" alt="image" src="https://github.com/filecoin-project/community/assets/21000523/40912964-13cd-488e-9a19-6f886623afe2">

Source: Estimates derived from [this blog](https://filecoin.io/blog/filecoin-circulating-supply/) and [Starboard](https://dashboard.starboard.ventures/circulating-supply)

# Predicted Outcomes

By alleviating the low MC/FDV concern for FIL, we anticipate a market shift in focus towards more economically meaningful metrics such as the daily inflation rate. This change would at the very least force investors to better anchor to estimates of the total supply based on the network's trajectory.

**Comment by jsoares** [comment]
I'll play devil's advocate. 

While this removes the FDV "issue", it doesn't help the supply inflation issue. So now people see high supply inflation _and_ no cap, meaning this can go on forever (blah-blah insider vesting, but does anyone want to hear that?). Unbounded supply sounds fine when printed next to [0.5% inflation](https://ultrasound.money/), but far less attractive when printed next to 26% inflation.

**Reply by jnthnvctr** [reply]
I think it's a valid flag. Imho the daily inflation (or even the yearly inflation) at least is an easier to contextualize stat with the other info about what the "final" supply is based on our current trajectory of onboarding / renewals. Basically if people do double click into this, i think its easier to at least try and then supplement this info with the projections of what our implied total supply might be in the same view. 

Fwiw I think in general its worth us discussing how we can reshape the tokenomics to better incentivize the things that will bring sustainable value (paid deals, settling on chain) into the ecosystem (and in a non-gameable way) - but I know there's a lot of conflicting opinions here and that might be a deeper discussion.  

Imho this is still a positive step forward (if somewhat incremental) - but hopefully itll at the very least it can show that we aren't fully deadlocked as an eco and can make progress.

**Reply by Schwartz10** [reply]
@jsoares ignoring perceptions of outsiders, do you genuinely believe capping supply at 2B is the best tokenomic decision for Filecoin? We should be making small iterations towards the best end state, and from my perspective, capping supply for a blockchain ecosystem like Filecoin doesn't make much sense in the context of the value proposition of filecoin.

IF investors start getting cold feet because they think this has uncapped inflation, you could solve that problem separately.

If you do believe 2b cap still makes sense, i'm curious to hear your reasoning

**Reply by jsoares** [reply]
I don't think I said or even suggested anything in that direction.

What I did say is that the reasoning for this change is weak. From the perspective of a naive outsider, <high inflation, uncapped supply> seems strictly worse than <high inflation, capped supply>. These are not independent problems that you can tackle separately; the stated goal is to improve perception, but it's actually worsening it. The argument that we'll somehow be able to explain away the inflation despite not being able to explain away the FDV in also not persuasive. 

That said, I think the proposed change makes sense if argue on intrinsic reasons, e.g. suitability to the ecosystem and economic model of the network, rather than on investor perception.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
123848775475755 this is the optimal supply. 

i would buy more FIL if this would be the total, max supply. 

does that change the casinos rules or dashboard flashyflash numbers? possibly, possibly not. 

I talk to people and they say filecoin doesn't produce enough value to make up for the burden of printing half'ish of the supply upfront *shrug emoji*. The ProducedValueToPreprintedCoins ratio (PVTPCR) sucks. I propose we start creating value within the ecosystem instead of doing casino magic...

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
off topic, but shouldn't this live in the FIPs repo? same for prop 2

**Comment by Fatman13** [comment]
uncap baseline minting then base it on what schedule after the initial 770m? Would the existing baseline minting keep working after that? And how does it work once simple minting is depleted?

**Reply by anorth** [reply]
@jnthnvctr I think there's a worthwhile motivation behind your proposal. I don't think the 2bil token meme is doing us any favours. But it's lacking detail for people to game out what it all means. This seems related also to #887, which had a similar problem.

I think it would help to provide an explicit specification for the alternate minting function that you imagine, or at least one to use as a starting point.

**Reply by jnthnvctr** [reply]
So I think there's a few variations - I think all solve for breaking the 2b meme (though maybe with differing degrees / complexity):

- One set of folks I talked to proposed simply changing the APIs that report on "total supply" (CMC / the FIL specific equiv) and not changing the protocol itself - so that it does not explicitly state a max supply. I think if part of what we're trying to kill is the FDV meme - this is maybe the simplest, while maintaining the fewest number of changes. 

- An alternative would be to change baseline minting so [M_infB](https://spec.filecoin.io/systems/filecoin_token/block_reward_minting/) so its a function (perhaps for simplicity it could be a function where if RBP is under 1YiB its 770, and above 1YiB of RBP we have a function that has a small amt of emissions (e.g. 0.5% of supply) based on time). Then I think you get a constantly increasing supply only in the case we have constantly doubling hw, but we also don't have a max supply.

**Comment by helloivanco** [comment]
In addition to the proposal to uncap the baseline minting supply, I would like to suggest two further enhancements:

**1.	Reduce Half-Life from 6 Years to 4 Years**

Accelerating the token release schedule will help reduce the perceived overhang of future token releases, aligning the circulating supply more closely with the network’s current utility and value. This adjustment can enhance market confidence and increase liquidity by making the tokenomics more transparent and predictable for investors.

**2.	Implement Adjustable Burn Rate**

To counteract the inflationary pressures from an accelerated token release schedule and the uncapped baseline minting, implementing a dynamic burn rate could be beneficial. This mechanism will adjust based on network conditions and performance metrics, ensuring a balanced and sustainable token supply.

**Comment by jennijuju** [comment]
Transferred from community discussion to here per author's request

**Reply by zbg-ft** [reply]
After looking around, it's just more arguments! There's a famous saying among Chinese strategists: "Those who share the same desire win!" The most effective short-term solution is to temporarily suspend the release by Protocol Labs and the Filecoin Foundation, which will immediately have a "halving" effect! Let's work together in the same boat to create ecological prosperity! The Satoshi Nakamoto wallet has never been moved, a tribute to Satoshi Nakamoto!

**Reply by zbg-ft** [reply]
看了一圈，只是更多的争论！中国兵家有句名言：上下同欲者胜！短期最有效的办法是暂停协议实验室和filecoin基金会的释放，马上起到“减半”效应！同舟共济，缔造生态繁荣！中本聪钱包从未移动，向中本聪致敬！

**Comment by zbg-ft** [comment]
After four years of market baptism, Filecoin continues to be viewed pessimistically by the market! WHY? But this is the reality of the market! If you have the ability, reverse it! The "halving" makes Filecoin scarce, everything is actually for this goal, the rest is a waste!
